tools/underwater-gps.py

```diff
@@ -117,6 +117,7 @@ def format(message, now=0, lat=0, lon=0, orientation=0):
 
 sockit = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
 sockit.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+server.setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
```

Comment:
server?

Reply:
this actually doesn't belong, we don't want this one to broadcast